esmvalcore/preprocessor/_io.py

```diff
@@ -269,3 +281,88 @@ def _write_ncl_metadata(output_dir, metadata):
     write_ncl_settings(info, filename)
 
     return filename
+
+
+def _concatenate_overlapping_cubes(cubes):
+    """Concatenate time-overlapping cubes (two cubes only)."""
+    # we arrange [cube1, cube2] so that cube1.start <= cube2.start
+    if cubes[0].coord('time').points[0] <= cubes[1].coord('time').points[0]:
+        cubes = [cubes[0], cubes[1]]
+        logger.debug(f"Will attempt to concatenate cubes %s "
+                     f"and %s in this order", cubes[0], cubes[1])
+    else:
+        cubes = [cubes[1], cubes[0]]
+        logger.debug(f"Will attempt to concatenate cubes %s "
+                     f"and %s in this order", cubes[1], cubes[0])
+
+    # get time end points
+    time_1 = cubes[0].coord('time')
+    time_2 = cubes[1].coord('time')
+    data_start_1 = time_1.cell(0).point
+    data_start_2 = time_2.cell(0).point
+    data_end_1 = time_1.cell(-1).point
+    data_end_2 = time_2.cell(-1).point
+
+    # case 1: both cubes start at the same time -> return longer cube
+    if data_start_1 == data_start_2:
+        if data_end_1 < data_end_2:
+            logger.debug(f"Both cubes start at the same time but cube %s "
```

Comment:
```suggestion
            logger.debug("Both cubes start at the same time but cube %s "
```
no need to make an f-string if you're using it as a normal string